arviz/plots/lmplot.py

```diff
@@ -0,0 +1,334 @@
+"""Plot regression figure."""
+import warnings
+from numbers import Integral
+
+import xarray as xr
+import numpy as np
+from xarray.core.dataarray import DataArray
+
+from ..sel_utils import xarray_var_iter
+from ..rcparams import rcParams
+from .plot_utils import default_grid, filter_plotters_list, get_plotting_function
+
+
+def plot_lm(
+    y,
+    idata=None,
+    x=None,
+    y_model=None,
+    y_hat=None,
+    num_pp_samples=50,
+    kind_pp="samples",
+    kind_model="lines",
+    xjitter=False,
+    plot_dim=None,
+    backend=None,
+    y_kwargs=None,
+    y_hat_plot_kwargs=None,
+    y_hat_fill_kwargs=None,
+    y_model_plot_kwargs=None,
+    y_model_fill_kwargs=None,
+    backend_kwargs=None,
+    show=None,
+    figsize=None,
+    textsize=None,
+    axes=None,
+    legend=True,
+    grid=True,
+):
+    """Posterior predictive and mean plots for regression-like data.
+
+    Parameters
+    ----------
+    y : str or DataArray or ndarray
+        If str, variable name from observed_data
+    idata : InferenceData, Optional
+        Optional only if y is not str
+    x : str, tuple of strings, DataArray or array-like, optional
+        If str or tuple, variable name from constant_data
+        If ndarray, could be 1D, or 2D for multiple plots
+        If none, coords name of y (y should be DataArray).
+    y_model : str or Sequence, Optional
+        If str, variable name from posterior.
+        Its dimensions should be same as y plus added chains and draws.
+    y_hat : str, Optional
+        If str, variable name from posterior_predictive.
+        Its dimensions should be same as y plus added chains and draws.
+    num_pp_samples : int, Optional, Default 50
+    kind_pp : {"samples", "hdi"}, Default "samples"
+    kind_model : {"lines", "hdi"}, Default "lines"
+    plot_dim : str, Optional
+        Necessary if y is multidimensional.
+    backend : str, Optional
+        Select plotting backend {"matplotlib","bokeh"}. Default "matplotlib".
+    y_kwargs : dict, optional
+        Passed to :meth:`mpl:matplotlib.axes.Axes.plot` in matplotlib
+        and :meth:`bokeh:bokeh.plotting.figure.Figure.circle` in bokeh
```

Comment:
according to sphinxinv, the objects in bokeh docs containing circle are these: 

```
:py:class:`bokeh.models.glyphs.Circle`
:py:attribute:`bokeh.models.glyphs.Circle.x`
:py:attribute:`bokeh.models.glyphs.Circle.y`
:py:function:`bokeh.models.markers.Circle`
:py:function:`bokeh.models.markers.CircleX`
:py:function:`bokeh.models.markers.CircleY`
:py:method:`bokeh.plotting.Figure.circle`
:py:method:`bokeh.plotting.Figure.circle_x`
:py:method:`bokeh.plotting.Figure.circle_y`
:py:method:`bokeh.plotting.GMap.circle`
:py:method:`bokeh.plotting.GMap.circle_x`
:std:label:`bokeh.models.glyphs.circle`
:std:label:`bokeh.models.markers.circle`
:std:label:`bokeh.models.markers.circledot`
:std:label:`bokeh.models.markers.circlex`
:std:label:`bokeh.models.markers.circley`
```

the one you used is not an option, maybe it was in an old version?